### PR TITLE
docs: Add diagram images and update performance doc

### PR DESCRIPTION
**Pull request by ikashifullah**
## Overview

> Describe your change in detail
- update the performance topic with new content.
- add new performance diagrams
- update _we_ with _TELUS_

**Forum Issue #** `telus/technology-forum#0`
> Link to the [Technology Forum][technology-forum] where this issue was discussed _(optional)_.

### Details

> e.g.
> - add something to existing topic
> - create new topic
> - adjust grammar & typos

---

#### Meta

> Please read and confirm each of the following:

- [ ] this topic was discussed in the [Technology Forum][technology-forum] _(ignore if the pull request represents small changes)_
- [x] provided a descriptive topic and overview of contribution
- [x] documentation format follows the [topic template][template]
- [x] branch is up to date with the `main` branch
- [x] "work in progress" commits are squashed _(Hint: ["Squashing Commits"][guide-squash])_
- [x] commits follow the [Conventional ChangeLog][conventional-changelog] format
- [x] no sensitive content included, such as:
  - content considered competitive intelligence
  - security & privacy policy violating content
  - keys, tokens or credentials

[template]: https://github.com/telus/reference-architecture/blob/master/.template.md
[conventional-changelog]: https://github.com/conventional-changelog/conventional-changelog/tree/master/packages/conventional-changelog-angular
[guide-squash]: https://git-scm.com/book/id/v2/Git-Tools-Rewriting-History
[technology-forum]: https://github.com/telus/technology-forum]

### CODE REVIEW

File: performance/performance-overview.md [hunk 1]
```diff
@@ -20,6 +20,28 @@ At present, we largely monitor three Lighthouse performance metrics and associat
 
 ## How
 
+### Performance Measurement
+
+#### Web performance model
+
+- Web performance could be broken down into sequential time steps, which combine together.
```
Comment (gonzalovazquez) [inline performance/performance-overview.md:27]: It feels like this sentence is missing the how they are combined together. I think taking the sentence below and combining it for more coherence.

File: performance/performance-overview.md [hunk 2]
```diff
@@ -20,6 +20,28 @@ At present, we largely monitor three Lighthouse performance metrics and associat
...
+- T4 or client-side processing is the most complex of all; it requires optimizing critical rendering path.
+
+#### Implementation advantage
```
Comment (gonzalovazquez) [inline performance/performance-overview.md:30]: Implementation advantage of what?

File: performance/performance-overview.md [hunk 3]
```diff
@@ -20,6 +20,28 @@ At present, we largely monitor three Lighthouse performance metrics and associat
...
+
+- Clear division of responsibilities based on sequential breakdown.
+- Budget allocation
```
Comment (gonzalovazquez) [inline performance/performance-overview.md:33]: In term of performance, budget refers to the cost of transaction from either system (T1...T4). The more responsive is T1, the more performance budget the other systems have. This needs to be explained when mentioning Budget Allocation.

File: performance/performance-overview.md (hunk 2 above)
Comment (ikashifullah) [reply performance/performance-overview.md]: > assets/web-performance-model1.png - could you please convert the idea to text instead? I'm not sure if we can use this source.
> 
> assets/Telus-implementation-model.png - can you please abstract out ideas and make them more generic, like load balancer, applications (Front End/Back End or UI/API)?

Done, can you please verify again, thanks.

File: performance/performance-overview.md [hunk 4]
```diff
@@ -1,52 +1,74 @@
...
+To support teams in achieving performance excellence throughout telus.com, TELUS principally relies on a custom implementation of [Google Lighthouse](https://developers.google.com/web/tools/lighthouse/) for automated performance testing.
 
-Lighthouse is a framework for testing web pages against a host of performance best practices. When run, it provides detailed scoring across several KPIs along with actionable insights into what we're doing well and areas for improvement.
+Lighthouse is a framework for testing web pages against a host of performance best practices. Running it provides detailed scoring across several KPIs, along with actionable insights into what is being done well and areas for improvement.
 
-At present, we largely monitor three Lighthouse performance metrics and associated targets:
+At present, TELUS largely monitors these three Lighthouse performance metrics and associated targets:
 
 - Overall Performance Score >= 70
 - First Meaningful Paint < 3 seconds
```
Comment (gonzalovazquez) [inline performance/performance-overview.md:18]: We should add `Largest Contentful Paint < 3 seconds` as it is another metric we capture.

**Review by gonzalovazquez — APPROVED** [review]

**Comment by ikashifullah** [comment]
> assets/web-performance-model1.png - could you please convert the idea to text instead? I'm not sure if we can use this source.
> 
> assets/Telus-implementation-model.png - can you please abstract out ideas and make them more generic, like load balancer, applications (Front End/Back End or UI/API)?

Already done, new images are being drawn from scratch and updated. please check and confirm.

File: performance/performance-overview.md (hunk 4 above)
Comment (ikashifullah) [reply performance/performance-overview.md:18]: It's added there, please re-check. thanks

**Comment by ikashifullah** [comment]
Mistakenly, closed the PR, apologies.

**Comment by ikashifullah** [comment]
> assets/web-performance-model1.png - could you please convert the idea to text instead? I'm not sure if we can use this source.
> 
> assets/Telus-implementation-model.png - can you please abstract out ideas and make them more generic, like load balancer, applications (Front End/Back End or UI/API)?

We have converted the images into diagrams (textual), so you can re-check and confirm.

File: performance/performance-overview.md (hunk 2 above)
Comment (ikashifullah) [reply performance/performance-overview.md]: @czgraggen could you please add more details connecting with @gonzalovazquez if possible.

File: performance/performance-overview.md [hunk 5]
```diff
@@ -1,52 +1,75 @@
...
 
-TELUS-Lighthouse is configured to run in our build pipeline and, optionally, with Git hooks (e.g. pre-push). Thresholds can be enabled and modified, if desired by teams, to ensure underperforming web pages fail the build, or in the case of git hooks, prevent merging into master altogether.
+TELUS-Lighthouse is configured to run in the build pipeline, and optionally, with Git hooks (e.g. pre-push). Thresholds can be enabled and modified by teams to ensure underperforming web pages fail the build, or in the case of git hooks, prevent merging into Master altogether.
 
-TELUS-Lighthouse is also run against all customer-facing URLs each day; the results of which are surfaced in our internal analytics dashboards.
+TELUS-Lighthouse is also run against all customer-facing URLs each day, and these results are surfaced in TELUS's internal analytics dashboards.
 
-For additional performance reporting, we also encourage the use of [New Relic](https://newrelic.com/products/application-monitoring).
+For additional performance reporting, the use of [New Relic](https://newrelic.com/products/application-monitoring) is encouraged.
```
Comment (gonzalovazquez) [inline performance/performance-overview.md:54]: Can you change this to DynaTrace?

Comment (ikashifullah) [reply performance/performance-overview.md]: Updated.

**Review by gonzalovazquez — APPROVED** [review]

**Pull request merged**